src/app/kvm/views/LXDClusterDetails/LXDClusterVMs/LXDClusterVMs.test.tsx

```diff
@@ -31,59 +26,6 @@ describe("LXDClusterVMs", () => {
     jest.spyOn(reduxToolkit, "nanoid").mockReturnValue("123456");
   });
 
-  it("can get resources for a cluster VM", () => {
```

Comment:
This test seems to have been removed. Is this intentional?

Reply:
Yeah, that test was pretty much just testing an internal function call within the component